website/content/en/docs/best-practices/common-recommendation.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Common recommendations and suggestions"
+linkTitle: "Common suggestions"
+weight: 2
+description: Common recommendations and suggestions to built solutions with Operator SDK
+---
+
+## Overview
+
+Any recommendations or good practices suggested by the Kubernetes community, such as to develop [Operator pattern][operator-pattern] solutions or to use [controller-runtime][controller-runtime], also are good recommendations for those who are looking to build the operator projects with SDK. Also, see [Operator Best Practices][operator-best-practices].
+
+## Common Recommendations
+
+### Develop idempotent reconciliation solutions
+
+See that following the operator's pattern. You will create controllers, from [controller-runtime][controller-runtime], with a reconcile function responsible for synchronizing the resources until there's desired state in the cluster. So, it is recommended to develop idempotent solutions for your reconciliations.
+
+### Understand the API's concept
```

Comment:
I think mentioning this is specifically about Kubernetes API would be good. Maybe something like

## Understand Kubernetes APIs

Usually, an operator will consist of an a controller which is responsible for the reconciliation of a [Kubernetes API extension from a Custom Resource Definition.][k8s-crd-doc] Check the Kubebuilder documentation [Groups and Versions and Kinds, oh my!][kb-gkv] to get an idea of how to design better solutions.

Reply:
solved as we chat bout tks for the help.